specification/types.md

```diff
@@ -49,6 +49,17 @@ A structure which contains a subset of the fields defined in the `evaluation det
 - reason (string, optional)
```

Comment:
This actually is already explicitly required to be a string. So in reality this PR is editorial (it doesn't actually change requirements, just clarifies).